Compiler/fmem.h

```diff
@@ -19,16 +19,16 @@
 struct FMEM
 {
   char *data;
-  long len;                     // length of data in array
-  long size;                    // size of data array allocated
-  long magic;
-  long pos;                     // current seeked position
+  size_t len;                  // length of data in array
+  size_t size;                 // size of data array allocated
+  int magic;
+  int pos;                     // current seeked position
```

Comment:
pos also should be size_t, as it should be comparable to size and len fields.

Reply:
Done